content/ui-patterns/messaging.mdx

```diff
@@ -7,25 +7,112 @@ import {Box} from '@primer/react'
 
-- _Toasts or flash alerts are best suited for this type of messaging._
+(image here)
 
-**Highlights**: bring attention to how and why to engage with a feature
+**Feedback**: Communicates the result of a user action (submitting a form, toggling a setting, etc.) Feedback can be positive, negative, or a warning. Feedback is shown in either a **Banner** or **Inline message**. For forms, an inline message is used for individual field validation which may be combined with a banner if multiple fields have errors. Banners are placed at the top of the page or section they are related to, but still within the body content. Inline messages are placed near the action they are related to.
 
-- _Popovers are best suited for this type of messaging._
+(image here)
 
-**Updates**: inform the user of necessary updates or changes to the product as it relates to them.
+**Awareness**: Communicates information that is relevant to the user but not necessarily related to an action they've taken. This type of message is typically used to provide context or additional information about a feature or product. A **Callout** might be used to suggest an action or helpful tip, while a **Popover** is typically used to announce a new feature.
 
-- _Flash alerts are best suited for this type of messaging._
+(image here)
+
+## Message states
+
+<img
+  width="960"
+  alt="A group of labels showing the proper icon and color combination for each message state"
+  src="https://github.com/primer/css/assets/18661030/41d3a706-9aa3-4f05-8bad-5176f49a7d5d"
+/>
+
+### Info
+
+Use the `info` state to highlight messaging that may help the user complete a task or provide additional context. This state is typically used for informational purposes and is not critical to the user's experience.
+
+Examples:
+- a setting that impacts the current view
+- nudge towards a new feature
+- a non blocking action to unlock additional features
+
+### Warning
+
+Use the `warning` state to inform users of a potential issue or action that may impact their experience.
+
+Examples:
+- an upcoming account setting will become required soon
+- a degraded experience(link) due to connectivity or GitHub availability
+- a non destructive action is about to take place
+
+### Success
+
+Use the `success` state to notify users that the results of an action were successful.
+
+Examples:
+- saving an account setting if its otherwise not clear the save has taken place
+- an issue has been transferred successfully
+
+### Unavailable
+
+Reserve the `unavailable` state for **inline** system related degraded experiences (see docs for more information).
+
+Examples:
+- a row of a data table could not be loaded
+- an item in a list could not be loaded
+
+### Critical
+
+Use the `critical` state to notify users that the results of an action were unsuccessful, or to inform that an action is destructive. This state is typically used for form errors and confirmation dialogs.
```

Comment:
```suggestion
Use the `critical` state to notify users that the results of an action were unsuccessful, to display form errors, or to inform that an action is destructive.
```

I feel this removes some duplication as _confirmation dialogs_ would have to be specified more which would be about destructive actions again.